paddle/pybind/pybind.cc

```diff
@@ -116,6 +119,47 @@ PYBIND11_PLUGIN(core) {
         return self.data<float>()[offset];
       });
 
+  py::class_<LoDTensor>(m, "LoDTensor", R"DOC(LoD(Leval of Ddetails) Tensor.
+
+The tensor and LoD info should be created before creating the LoDTensor, then
+call the set_tensor and set_lod functions to set them.
+
+)DOC")
+      .def("set_tensor",
```

Comment:
these two interfaces are too trivial, add another construction function may be better, `__init__(lod, tensor)` because LoDTensor is a complete concept, not a union of two different concepts (though implementation is, but that should be hidden from user)

Reply:
Yes, in Paddle, we have a DataConverter to convert user's data into argument. 
Whether should we implement a DataConverter or just expose a user-friendly interface in LoDTensor.

Reply:
@Superjom Add `__init__` construction function in pybind. But I think the `set_tensor` and `set_lod` is still needed. Since we usually get empty LoDTensor from the variable, then we need to set tensor and lod for this LoDTensor, and some times we only need to set tensor for the case without lod info. So the `set_tensor` and `set_lod` are stilled reserved.

Reply:
@QiJune You are right. We should provide converter to get lod for the sequence inputs. The inputs are a list of list or a list of numpy array. Maybe it's relatively convenient to convert in Python API instead of pybind or C++ code.  @reyoung @Superjom